### PR TITLE
Add various declared license mapping

### PR DESCRIPTION
**Pull request by tsteenbe**
See individual commits for details.

### CODE REVIEW

File: utils/spdx/src/main/resources/declared-license-mapping.yml [hunk 1]
```diff
@@ -550,4 +551,4 @@
 "zope 2.0": ZPL-2.0
 "zope 2.1": ZPL-2.1
 "zope license": ZPL-2.1
-"zope v2.1": ZPL-2.1
+"zope v2.1": ZPL-2.1
```
Comment (fviernau) [inline utils/spdx/src/main/resources/declared-license-mapping.yml]: This is unrelated and should be dropped.

File: utils/spdx/src/main/resources/declared-license-mapping.yml [hunk 2]
```diff
@@ -552,4 +553,4 @@
 "zope 2.0": ZPL-2.0
 "zope 2.1": ZPL-2.1
 "zope license": ZPL-2.1
-"zope v2.1": ZPL-2.1
\ No newline at end of file
+"zope v2.1": ZPL-2.1
```
Comment (fviernau) [inline utils/spdx/src/main/resources/declared-license-mapping.yml:556]: The fix-up went into the wrong commit.

**Review by fviernau — CHANGES_REQUESTED** [review]

File: utils/spdx/src/main/resources/declared-license-mapping.yml (hunk 2 above)
Comment (tsteenbe) [reply utils/spdx/src/main/resources/declared-license-mapping.yml]: I helps if you do a `git push -f` after you fix up things :-(

**Review by fviernau — APPROVED** [review]
